showup/migrations/0004_merge_20191118_1643.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 2.2.5 on 2019-11-18 21:43
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("showup", "0002_auto_20191118_1537"),
+        ("showup", "0003_auto_20191118_1619"),
+    ]
+
+    operations = []
```

Comment:
What does this migration do? I suspect that it does nothing because "operations" is empty, but I'm not sure.